src/main/java/io/github/perrymant/moneymaker/AccountRepository.java

```diff
@@ -0,0 +1,37 @@
+package io.github.perrymant.moneymaker;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+
+
+class AccountRepository {
+
+    Account read(String fileName) {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
We don't need to create a new one of these each time. Just keep one as a field and reuse here and in `write`